### PR TITLE
https://issues.redhat.com/browse/ACM-9743 RHACM Errata 2.9.3 (needs comments, deadline 3/11)

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-9743

### CODE REVIEW

File: release_notes/errata.adoc [hunk 1]
```diff
@@ -7,6 +7,38 @@ By default, Errata updates are automatically applied when released. The details
 
 See link:../install/upgrade_hub.adoc#upgrading-by-using-the-operator[Upgrading by using the operator] for more information about upgrades.
 
+== Errata 2.9.3
+
+//potential issues that can be listed, but awaiting confirmation; these are the issues listed as public from the advisory Summary tab
+
+* link:https://issues.redhat.com/browse/ACM-8728[ACM-8728]
```
Comment (cadawson) [inline release_notes/errata.adoc]: Removed

File: release_notes/errata.adoc [hunk 2]
```diff
@@ -7,6 +7,38 @@ By default, Errata updates are automatically applied when released. The details
...
+
+* link:https://issues.redhat.com/browse/ACM-8998[ACM-8998]
```
Comment (cadawson) [inline release_notes/errata.adoc]: Removed

File: release_notes/errata.adoc [hunk 3]
```diff
@@ -7,6 +7,38 @@ By default, Errata updates are automatically applied when released. The details
...
+
+* link:https://issues.redhat.com/browse/ACM-9123[ACM-9123]
```
Comment (cadawson) [inline release_notes/errata.adoc]: * link:https://issues.redhat.com/browse/ACM-9123[ACM-9123]: This fixes a bug that prevented upgrading some imported clusters to {product-title-short} version 2.9.

File: release_notes/errata.adoc [hunk 4]
```diff
@@ -7,6 +7,38 @@ By default, Errata updates are automatically applied when released. The details
...
+
+* link:https://issues.redhat.com/browse/ACM-9145[ACM-9145]
```
Comment (cadawson) [inline release_notes/errata.adoc]: * link:https://issues.redhat.com/browse/ACM-9145[ACM-9145]: This fixes an issue that prevented the upgrade of imported clusters by using the `ClusterCurator`.

File: release_notes/errata.adoc [hunk 5]
```diff
@@ -7,6 +7,38 @@ By default, Errata updates are automatically applied when released. The details
...
+
+* link:https://issues.redhat.com/browse/ACM-9186[ACM-9186]
```
Comment (cadawson) [inline release_notes/errata.adoc]: Removed

File: release_notes/errata.adoc [hunk 6]
```diff
@@ -7,6 +7,38 @@ By default, Errata updates are automatically applied when released. The details
...
+
+* link:https://issues.redhat.com/browse/ACM-9311[ACM-9311]
```
Comment (cadawson) [inline release_notes/errata.adoc]:
```suggestion
* link:https://issues.redhat.com/browse/ACM-9311[ACM-9311]: Hosted Control Planes in {mce} version 2.9 is only compatible with {ocp} version 4.14. 
```

File: release_notes/errata.adoc [hunk 7]
```diff
@@ -7,6 +7,38 @@ By default, Errata updates are automatically applied when released. The details
...
+
+* link:https://issues.redhat.com/browse/ACM-9467[ACM-9467]
```
Comment (cadawson) [inline release_notes/errata.adoc]: Removed. It has a replacement issue.

File: release_notes/errata.adoc [hunk 8]
```diff
@@ -7,6 +7,38 @@ By default, Errata updates are automatically applied when released. The details
...
+
+* link:https://issues.redhat.com/browse/ACM-9719[ACM-9719]
```
Comment (cadawson) [inline release_notes/errata.adoc]: Removed

File: release_notes/errata.adoc [hunk 9]
```diff
@@ -7,6 +7,38 @@ By default, Errata updates are automatically applied when released. The details
...
+
+* link:https://issues.redhat.com/browse/ACM-9724[ACM-9724]
```
Comment (cadawson) [inline release_notes/errata.adoc]: Removed

File: release_notes/errata.adoc [hunk 10]
```diff
@@ -7,6 +7,38 @@ By default, Errata updates are automatically applied when released. The details
...
+
+* link:https://issues.redhat.com/browse/ACM-9746[ACM-9746]
```
Comment (cadawson) [inline release_notes/errata.adoc]: No summary

File: release_notes/errata.adoc [hunk 11]
```diff
@@ -7,6 +7,38 @@ By default, Errata updates are automatically applied when released. The details
...
+
+* link:https://issues.redhat.com/browse/ACM-9885[ACM-9885]
```
Comment (cadawson) [inline release_notes/errata.adoc]: Removed

File: release_notes/errata.adoc [hunk 12]
```diff
@@ -7,6 +7,38 @@ By default, Errata updates are automatically applied when released. The details
...
+
+* link:https://issues.redhat.com/browse/ACM-9930[ACM-9930]
```
Comment (cadawson) [inline release_notes/errata.adoc]:
```suggestion
* link:https://issues.redhat.com/browse/ACM-9930[ACM-9930]: Resolves the case when a `PlacementBinding` misconfiguration blocked Policies from being updated until the misconfiguration was fixed. An example of this is when the wrong API group was specified in the `placementRef`. 
```

File: release_notes/errata.adoc [hunk 13]
```diff
@@ -7,6 +7,38 @@ By default, Errata updates are automatically applied when released. The details
...
+
+* link:https://issues.redhat.com/browse/ACM-9947[ACM-9947]
```
Comment (cadawson) [inline release_notes/errata.adoc]: Removed

**Review by cadawson — COMMENTED** [review]
This needs some summaries in the issues.

File: release_notes/errata.adoc (hunk 10 above)
Comment (oafischer) [reply release_notes/errata.adoc]: Maybe: Fixes an issue that caused the search collector to create too many logs when deploying Argo CD and Argo applications to a managed cluster.

**Review by oafischer — APPROVED** [review]
Looks good once the missing summary is added. Thanks!

**Review by rhaignere — APPROVED** [review]
Reviewed and approved. Thanks!

**Pull request merged**